vortex/core/factory.py

```diff
@@ -348,6 +352,16 @@ def create_dataloader(dataloader_config : EasyDict,
         collater_args['dataformat'] = dataset.data_format
         collate_fn = create_collater(collate_fn, **collater_args)
 
+        # Re-initialize dataset (Temp workaround), adding `disable_image_auto_pad` to collate_fn object 
+        # to disable auto pad augmentation
+        if hasattr(collate_fn,'disable_image_auto_pad'):
```

Comment:
would it be simpler if the `create_dataset` only called once? by specifying a variable

Reply:
any suggestion? so, create_dataset is called before collater because collater need data format from dataset. But the disable args is from collater si this is like a cycle and i havent found any better approach for this

Reply:
ohh yaa I forgot the collater needs data format from dataset